test/integration/eu-west-1/k8s-qa/services/terragrunt.hcl

```diff
@@ -144,35 +163,35 @@ inputs = {
   atlantis_webhook_events      = ["issue_comment", "pull_request", "pull_request_review", "push"]
   atlantis_chart_version       = "4.1.2"
 
-  atlantis_flux_repo_name     = "platform-manifests-qa"
-  atlantis_flux_repo_owner    = "dfds"
-  atlantis_flux_repo_branch   = "main"
+  atlantis_flux_repo_name   = "platform-manifests-qa"
+  atlantis_flux_repo_owner  = "dfds"
+  atlantis_flux_repo_branch = "main"
 
   # --------------------------------------------------
   # Blackbox Exporter
   # --------------------------------------------------
 
-  blackbox_exporter_deploy              = "true"
-  blackbox_exporter_github_owner        = "dfds"
-  blackbox_exporter_repo_name           = "platform-manifests-qa"
-  blackbox_exporter_repo_branch         = "main"
-  blackbox_exporter_monitoring_targets  = [
+  blackbox_exporter_deploy       = "true"
+  blackbox_exporter_github_owner = "dfds"
+  blackbox_exporter_repo_name    = "platform-manifests-qa"
+  blackbox_exporter_repo_branch  = "main"
+  blackbox_exporter_monitoring_targets = [
     {
-      "name"    = "example"
-      "url"     = "https://example.com/"
-      "module"  = "http_2xx"
+      "name"   = "example"
+      "url"    = "https://example.com/"
+      "module" = "http_2xx"
     }
   ]
 
   # --------------------------------------------------
   # Helm Exporter
   # --------------------------------------------------
 
-  helm_exporter_deploy              = "true"
-  helm_exporter_helm_chart_version  = "1.2.3"
-  helm_exporter_github_owner        = "dfds"
-  helm_exporter_repo_name           = "platform-manifests-qa"
-  helm_exporter_repo_branch         = "main"
+  helm_exporter_deploy             = "true"
+  helm_exporter_helm_chart_version = "1.2.3"
```

Comment:
We should attempt 1.2.4 that was released last week

Reply:
This is just a change in whitespace, but I can put up another PR to put this change in separately.

Reply:
Extracted out to this PR: https://github.com/dfds/infrastructure-modules/pull/733